test/extensions/filters/http/wasm/test_data/BUILD

```diff
@@ -38,3 +38,11 @@ wasm_cc_binary(
         "@proxy_wasm_cpp_sdk//contrib:contrib_lib",
     ],
 )
+
+genrule(
```

Comment:
Could I get some bazel help here?

Reply:
`//bazel/foreign_cc:wavm` builds only the library, you'd need to add `cc_binary` target for the tool if you want to build it.

Reply:
it does build the binary (I checked).  I have been messing with this for a few hours and bazel is just not my thing.  I'm going to rip this out and we can add it in later.